### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by daniayivi** [validator/validator#1103]
le vérificateur HTML a mal identifié la langue de mon site,

**Comment by sideshowbarker** [comment]
Please provide the URL for the HTML document, or the HTML document itself.

**Comment by daniayivi** [comment]
view-source:www.portfoliodani.unaux.com

**Comment by sideshowbarker** [comment]
The problem is that the source of http://www.portfoliodani.unaux.com/ looks like this:

```html
<html><body><script type="text/javascript" src="/aes.js" ></script><script>function toNumbers(d){var e=[];d.replace(/(..)/g,function(d){e.push(parseInt(d,16))});return e}function toHex(){for(var d=[],d=1==arguments.length&&arguments[0].constructor==Array?arguments[0]:arguments,e="",f=0;f<d.length;f++)e+=(16>d[f]?"0":"")+d[f].toString(16);return e.toLowerCase()}var a=toNumbers("f655ba9d09a112d4968c63579db590b4"),b=toNumbers("98344c2eee86c3994890592585b49f80"),c=toNumbers("08d78f3f3cfd3ae25942b652bc9ddd19");document.cookie="__test="+toHex(slowAES.decrypt(c,2,a,b))+"; expires=Thu, 31-Dec-37 23:55:55 GMT; path=/"; location.href="http://www.portfoliodani.unaux.com/?i=1";</script><noscript>This site requires Javascript to work, please enable Javascript in your browser or use a browser with Javascript support</noscript></body></html>
```

See https://validator.w3.org/nu/?showsource=yes&doc=http://www.portfoliodani.unaux.com/

So the `<html>` start tag there has no `lang` attribute. It should have `<html lang="">` or `<html lang="fr">`

The reason that browsers show a different document is, that source apparently runs some JavaScript to load the document contents. But the checker doesn’t run that JavaScript — instead it loads that source as-is.
